### ISSUE TITLE
UI Buttons (mask, channel view, tile) missing.

### ISSUE THREAD

**Issue by Mrfunreal** [Sky-rym/VTFEdit-Reloaded#18]
One of the great things this version can do is show buttons that I can press to toggle the alpha mask, or tile the texture without having to first click on "View - mask".
However, those buttons disappear when you load VTFEditReloaded via double clicking  VTF file, so now we're back to using the uncomfortable keybind of Ctrl+M and such.

It would be nice if those buttons would be permanent, cause they're one of the two feautres i really wanted.

**Comment by WereTech** [comment]
It works for me, not sure why it wouldn't be working for you. Are you on Windows? I'm on Windows 10 right now.

**Comment by Mrfunreal** [comment]
I'm on Win10 too.

**Comment by WereTech** [comment]
Can you share a screenshot of what your vtfedit looks like when directly opening a file? I'm thinking of a case of you maybe not changing the file association to the newer one if you hadn't directly replaced the older version.

**Comment by Mrfunreal** [comment]
Sure, here you go.

![VTFEDIT_RELOADED](https://github.com/Sky-rym/VTFEdit-Reloaded/assets/62206882/c9b4826b-f8f4-4254-83ed-06288f6ff4e2)

**Comment by WereTech** [comment]
It's pointing to the wrong version of vtfedit, you need to change the exe it runs by default by doing Right Click > Open With > Then select the exe that's the latest version

**Comment by Mrfunreal** [comment]
I had done that, but it appears as if I had another version of the exe somewhere and it for some reason loaded the old version instead of the new one I told it to use.

I had switched default programs from old "VTFEdit Reloaded.exe" from the x86 folder to the new one that came without the x86 and x64 folders, but it kept using the old one for some reason. Presumably because windows does not like switching between two exes of the same name, but in different places.

So I deleted the old one from my system entirely, moved the new one to another spot. Now it works.
Thank you for your time and patience.
